docker/docker.go

```diff
@@ -108,8 +108,12 @@ func NewImage(qname, user, password string, insecureTLS bool) (*Image, error) {
 	if name == "" {
 		name = strings.Join(nameParts, "/")
 	}
+	if insecureRegistry {
+		registry = fmt.Sprintf("http://%s/v2", registry)
+	} else {
+		registry = fmt.Sprintf("https://%s/v2", registry)
```

Comment:
My test registry is without SSL. For invalid certificates, has been implemented in PR #19 .